installing/installing_ibm_z/installing-ibm-z-kvm.adoc

```diff
@@ -95,6 +95,14 @@ include::modules/ibm-z-secure-execution.adoc[leveloffset=+2]
 
 * link:https://www.ibm.com/docs/en/linux-on-systems?topic=ibmz-secure-execution[Linux as an IBM Secure Execution host or guest]
 
+include::modules/ibmz-configure-nbde-with-static-ip.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources_configure-nbde-ibm-z-kvm"]
+.Additional resources
+
+* xref:../../installing/install_config/installing-customizing.adoc#installing-customizing[Creating machine configs with Butane].
```

Comment:
```suggestion
* xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-butane_installing-customizing[Creating machine configs with Butane]
```
Please remove the period from end of the additional resources xrefs in these 4 assemblies as they are not complete sentences. I also updated the xref anchor to go directly to the "Creating machine configs with Butane" section which could be updated in all 4 assemblies. Thank you!